lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1321,7 +1296,8 @@ def deploy_vm(self):
         self.configure_cpu_and_memory(vm_obj=vm_obj, vm_creation=True)
         self.configure_disks(vm_obj=vm_obj)
         self.configure_network(vm_obj=vm_obj)
-        if self.should_deploy_from_template() and len(self.params['customizations']) > 0:
+
+        if vm_obj is None or self.should_deploy_from_teplate():
```

Comment:
I ask again: When this condition will be skipped ?
I understand: When we deploy from scratch or from template. There is an other case ?